.github/workflows/actions.yml

```diff
@@ -990,3 +990,80 @@ jobs:
       working-directory: ./android
       run: |
         gradle build
+
+  ubuntu-24-04-from-tarball-cxx-23:
```

Comment:
Nit: I think we have enough `from-tarball` configurations as is, since this one doesn't exactly need to prove that it builds specifically from a tarball we can speed feedback up by building from git clone instead to not block on tarball generation job.

Reply:
Done!